.scala-steward.conf

```diff
@@ -0,0 +1,12 @@
+updates.ignore = [
+  { groupId = "com.google.protobuf", artifactId = "protobuf-java" },
+  { groupId = "org.scalatest", artifactId = "scalatest" },
+  { groupId = "com.typesafe", artifactId = "config" },
+  { groupId = "com.typesafe", artifactId = "ssl-config-core" },
+  { groupId = "com.typesafe", artifactId = "sbt-mima-plugin" },
+  { groupId = "com.typesafe.sbt", artifactId = "sbt-multi-jvm" },
+  { groupId = "com.lightbend.paradox", artifactId = "sbt-paradox" },
+  { groupId = "com.lightbend.paradox", artifactId = "sbt-paradox-apidoc" },
+  { groupId = "com.lightbend.akka", artifactId = "sbt-paradox-akka" }
```

Comment:
isn't it good to keep paradox in, since it's often changes we do ourselves in paradox and we want to use the latest?